Composer/packages/client/src/CreationFlow/DefineConversation/index.js

```diff
@@ -109,7 +136,14 @@ export function DefineConversation(props) {
             />
           </StackItem>
         </Stack>
-        {enableLocationBrowse && <LocationSelectContent onChange={updateLocation} allowOpeningBot={false} />}
+        {enableLocationBrowse && (
+          <LocationSelectContent
+            updateCurrentPath={updateCurrentPath}
+            allowOpeningBot={false}
```

Comment:
If we don't have path for `true`, let's remove this prop and remove unneeded code paths.